src/C-interface/csr/bml_setters_csr_typed.c

```diff
@@ -97,7 +97,7 @@ void TYPED_FUNC(
  *
  *
  */
-void TYPED_FUNC(
+/*void TYPED_FUNC(
```

Comment:
Is there a reason to keep that old function commented out?

Reply:
Nope. Just forgot to clean that up. I'll do that now.